src/sklearn_evaluation/plot/classification.py

```diff
@@ -252,11 +254,13 @@ def _plot_cm(cm, cmap, ax, target_names, normalize):
 
 
 # http://scikit-learn.org/stable/auto_examples/ensemble/plot_forest_importances.html
+@SKLearnEvaluationLogger.log(feature='plot')
 def feature_importances(data,
                         top_n=None,
                         feature_names=None,
                         orientation='horizontal',
-                        ax=None):
+                        ax=None,
+                        **kwargs):
```

Comment:
do we still need the `**kwargs`? That's going to cause the issue I mentioned earlier. (users pass arbitrary arguments that have no effect)


also, can you check what happens if we have the `ax` argument? I think posthog will complain that it's not JSON serializable but we must ensure it doesn't break the library


```python
import matplotlib.pyplot as plt

ax = plt.gca()

some_plotting_function(..., ax=ax)

```